src/main/java/frc/robot/commands/BangBangShooter.java

```diff
@@ -155,7 +158,7 @@ else if (currentControlMode == ControlMethod.HOLD) {
   @Override
   public void end(boolean interrupted) {
     shooter.stopAll();
-    shooter.setLEDRing(false);
+    shooter.vision.setLEDRing(false);
```

Comment:
This should be shooter.vision.setMode

Reply:
In that case, what string should I use for the argument?